composer.json

```diff
@@ -17,7 +17,7 @@
         }
     ],
     "require": {
-        "php": "^5.3 || ^7.0",
+        "php": "^7.1",
```

Comment:
Because we use this bundle on some project that still are on PHP 5.6, if we need to contribute to this bundle we should support php5.6.
I know it's cool to develop on the last available version, but we don't always have the choice :/